templates/geosafe/analysis/create.html

```diff
@@ -520,7 +520,13 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
             });
 
             {# Zoom first to global world view #}
-            zoom_to_box(map, [0, 45, 0, -45]);
+            {%  if analysis and analysis.user_extent %}
+                user_extent_string = "{{ analysis.user_extent }}";
+                extent_after_run = user_extent_string.split(",");
+                zoom_to_box(map, extent_after_run);
```

Comment:
yes, i have done it.
`analysis.user_extent` is on the string data type.
we need to pass an array to the second argument of `zoom_to_box` function.
that's why i convert them to array first.

now i have another concern. the order of `user_extent` coordinates from the backend is puzzled me.
looking at the doc `geosafe/views/analysis.py`:
`# This bbox is in the format [x0,y0,x1,y1]`
do the coordinates order is correct @lucernae?  
currently, i have problem rendering `zoom_to_box` using the coordinates from the `user_extent`.